pkg/cmd/taskrun/log_reader.go

```diff
@@ -273,11 +279,25 @@ func (lr *LogReader) waitUntilPodNameAvailable(timeout time.Duration) (*v1alpha1
 			}
 			if first {
 				first = false
-				fmt.Fprintln(lr.Stream.Out, "Task still running ...")
+				fmt.Fprintln(lr.Stream.Out, "Waiting for logs to be available...")
```

Comment:
Updating message here as it can be misleading. For instance, if you run `tkn taskrun logs -f` after a task has failed, it would print `Task still running` even though it has already failed.